### PR TITLE
Change workspace structure for downloaded components (#463)

### PR DESCRIPTION
**Pull request by m00g3n**
- #463
- #450

See also: 
- https://github.com/kyma-incubator/reconciler/issues/527

### CODE REVIEW

File: pkg/reconciler/chart/component.go [hunk 1]
```diff
@@ -15,6 +15,10 @@ type Component struct {
 	configuration map[string]interface{}
 }
 
+func (c *Component) isRepository() bool {
```
Comment (tobiscr) [inline pkg/reconciler/chart/component.go:18]: Naming can be a bit confusing.. `component.isRepository()`  - maybe we could point out whether its an external component coming form a GIT repo, like `isExternalGitComponent` or something.

**Review by tobiscr — COMMENTED** [review]
Looks in general very good! Just one short comment

**Review by tobiscr — CHANGES_REQUESTED** [review]
Maybe we can use a bit more expressive name for `isRepository`.

**Review by suleymanakbas91 — APPROVED** [review]

**Pull request merged**